test/functional/dispatch-handle-register.c

```diff
@@ -81,10 +81,10 @@ void functional_dispatch_handle_register(UNUSED(void **state))
 
   info.con = MALLOC(struct connection);
   info.con->closed = true;
-  info.con->cc.pluginkey = pluginkey;
+  memcpy(info.con->cc.pluginkeystring, pluginkey, PLUGINKEY_STRING_SIZE);
```

Comment:
The pluginkeystring is an array and not of type `char *`. So copying seems legit and necessary.